frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -15,14 +15,17 @@
  */
 
 import styled from "@emotion/styled"
-import { keyframes } from "@emotion/react"
 import { transparentize } from "color2k"
 
 export interface StyledSidebarProps {
   isCollapsed: boolean
   sidebarWidth: string
 }
 
+export interface StyledSidebarNavButtonProps {
+  isExpanded: boolean
+}
+
```

Comment:
super nit: prefer this interface declaration to be right above its usage in `StyledSidebarNavButton` consistent with organization of the rest of the file

Reply:
Got it! Updated in [6919ec3](https://github.com/streamlit/streamlit/pull/6050/commits/6919ec3e85adc6a8573561634ed007127edb35e1)